numpy/testing/__init__.py

```diff
@@ -9,7 +9,13 @@
 
 from unittest import TestCase
 
-from . import decorators as dec
-from .nosetester import run_module_suite, NoseTester as Tester, _numpy_tester
-from .utils import *
+
+from ._private import decorators as dec
+from ._private.nosetester import run_module_suite, NoseTester as Tester, _numpy_tester
+from ._private.utils import __all__
+from ._private.utils import *
+
+__all__ = __all__[:]
+__all__.extend(['TestCase', 'run_module_suite'])
```

Comment:
There might be some controversy as to what should be in `__all__`.

Reply:
I think you can spell this `__all__ = _private.utils.__all__ + ['TestCase', 'run_module_suite']`

Reply:
Doesn't work unless `_private/__init__.py` is changed, `_private.utils` cannot be imported otherwise. One of various alternatives is `from ._private import utils as _utils` and go from there, but I don't think it is an improvement.

Reply:
> _private.utils cannot be imported otherwise

I don't agree. The `from ._private.utils import *` line is enough to make the `_private` member exist

Reply:
Hmm, when I tried it before, it didn't work. Now it does, so I must have neglected to clear something in the process.